webapp/graphite/render/functions.py

```diff
@@ -2820,8 +2820,10 @@ def reduceSeries(requestContext, seriesLists, reduceFunction, reduceNode, *reduc
         i = reduceMatchers.index(node)
         metaSeries[reduceSeriesName][i] = series
   for key in keys:
-    metaSeries[key] = SeriesFunctions[reduceFunction](requestContext,metaSeries[key])[0]
+    print metaSeries[key]
+    metaSeries[key] = SeriesFunctions[reduceFunction](requestContext,*[[l] for l in metaSeries[key]])[0]
     metaSeries[key].name = key
+    print metaSeries[key]
```

Comment:
Please remove these two stray print statements, rebase, and update this pr. They shouldn't be in committed code.